Tools/msi/lib/lib.wixproj

```diff
@@ -15,7 +15,7 @@
         <EmbeddedResource Include="*.wxl" />
     </ItemGroup>
     <ItemGroup>
-        <ExcludeFolders Include="Lib\test;Lib\tests;Lib\tkinter;Lib\idlelib;Lib\turtledemo" />
+        <ExcludeFolders Include="Lib\test;Lib\tests;Lib\tkinter;Lib\idlelib;Lib\turtledemo;Lib\turtle.py" />
```

Comment:
Actually, on second look at this, these need to be folders.

`$(PySourcePath)Lib\turtle.py` needs to be added after line 21

Reply:
done...